source/APILookupSoup.txt

```diff
@@ -0,0 +1,91 @@
+#
+# This file is part of gtkD.
+#
+# gtkD is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version, with
+# some exceptions, please read the COPYING file.
+#
+# gtkD is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with gtkD; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+#
+wrap: soup
+file: Soup-2.4.gir
```

Comment:
should we use soup 3 or later with gtk 4? (e.g. replaced SoupBuffer with Gio Bytes)

For shumate that requires `-Dlibsoup3=true` though

Reply:
I had to use soup 3 for the latest shumate version.